### PR TITLE
Feature/tsk 136 init users microservice

### PR DESCRIPTION
**Pull request by Hoocki**

### CODE REVIEW

File: user-manager/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	compileOnly {
+		extendsFrom(configurations.annotationProcessor.get())
+	}
+}
+
+dependencies {
+	compileOnly("org.projectlombok:lombok")
+	annotationProcessor("org.projectlombok:lombok")
+	implementation("org.springframework.boot:spring-boot-starter-web")
+	testImplementation("org.springframework.boot:spring-boot-starter-test")
```
Comment (adsukharev) [inline user-manager/build.gradle.kts:25]: it's better to separate dependencies by stages, like in this example:
```
    annotationProcessor("io.micronaut:micronaut-http-validation")
    annotationProcessor("org.projectlombok:lombok")
    annotationProcessor("io.micronaut.serde:micronaut-serde-processor")

    implementation("io.micronaut.serde:micronaut-serde-jackson")

    runtimeOnly("ch.qos.logback:logback-classic")
    runtimeOnly("org.yaml:snakeyaml")

    compileOnly("org.projectlombok:lombok")
    compileOnly("io.micronaut:micronaut-http-client")

    testImplementation("org.mockito:mockito-junit-jupiter")
    testImplementation("org.assertj:assertj-core")
    testImplementation("io.micronaut:micronaut-http-client")
```

File: user-manager/src/test/java/com/qpay/usermanager/UserManagerApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class UserManagerApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
+
+}
```
Comment (adsukharev) [inline user-manager/src/test/java/com/qpay/usermanager/UserManagerApplicationTests.java]: remove this test

**Review by adsukharev — APPROVED** [review]
1) your commit messages does not contains task id
2) Increase minimum number of approvals to 2
3) Change the permission to prevent pushing to master

**Comment by adsukharev** [comment]
@kostya4201337 clone this repo and run the app. You need to check that everything works right on your side. Also learn a bit about gradle, how to build project and run tests. If you need help, ask Roma

**Comment by adsukharev** [comment]
Task number must be in square brackets [TSK-1] your message

**Review by kostya4201337 — APPROVED** [review]

**Pull request merged**